4.0-MIGRATION-GUIDE.md

```diff
@@ -141,5 +169,43 @@ PRs:
 
 ---
 
+### Popper
+
+`Popper` was changed to a Functional Component with a forwarded ref. If you passed a `ref` object to
+`Popper` before, it will now point to the element rather than the `Popper` instance.
+
+PR: https://github.com/Workday/canvas-kit/pull/528
+
+---
+
+### Tooltip
+
+Tooltip now uses React Portals and has been completely updated to make attaching tooltips much
+easier.
```

Comment:
```suggestion
Tooltip now uses our Popper component and has been completely updated to make attaching tooltips
much easier.
```

Reply:
Should we add an example of the new usage?

Like:

```tsx
<Tooltip title="Add">
  <IconButton variant={IconButton.Variant.Circle} icon={xIcon} aria-label="Add" />
</Tooltip>
```

Reply:
Planning to convert this migration guide into a 4.0 guide before the final release with notes about all the new features. Definitely want to include this, but was thinking it belonged there 👍